sysdeps/linux/include/abi-bits/utsname.h

```diff
@@ -0,0 +1 @@
+../../../../abis/mlibc/utsname.h
```

Comment:
Linux should use the Linux ABI. The header should also be in the Linux abi-bits.